dashboards/jupyterhub.jsonnet

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env jsonnet -J ../vendor
+#!/usr/bin/env -S jsonnet -J ../vendor
```

Comment:
I had to use `-S` to not have env parse the content above as a command with spaces in it.